src/standard/expr.ml

```diff
@@ -841,7 +842,7 @@ module Ty = struct
   let add_tag (t : t) k v = t.ty_tags <- Tag.add t.ty_tags k v
   let add_tag_opt (t : t) k o = t.ty_tags <- Tag.add_opt t.ty_tags k o
   let add_tag_list (t : t) k l = t.ty_tags <- Tag.add_list t.ty_tags k l
-  
+
```

Comment:
spurious white line added here

Reply:
That wan't intentional, my editor automatically removes spaces at the ends of lines and that line only had two spaces on it so it removed them. I can put them back if you want :)

Reply:
oh, I didn't see that, of course removing trailing white-spaces is good, ^^